### ISSUE TITLE
Remap Opt/Alt+T hotkey to Opt/Alt+O

### ISSUE THREAD

**Issue by arouinfar** [phetsims/circuit-construction-kit-common#1217]
When testing in VoiceOver, using Opt+T to jump to the carousel doesn't work. Instead, it tries to do something with tables. The Opt+P hotkey to jump to the Construction Area works well. For proximity in QWERTY layouts, Opt+O is a good alternative to Opt+T, and doesn't appear to conflict with other screenreader keys.

@samreid please
- [ ] Remap Opt/Alt+T hotkey to Opt/Alt+O
- [ ] Update Keyboard Shortcuts Dialog accordingly, including description strings
